tensorflow/python/keras/losses.py

```diff
@@ -1843,15 +1843,35 @@ def deserialize(name, custom_objects=None):
 
 @keras_export('keras.losses.get')
 def get(identifier):
-  """Retrieves a Keras loss function.
+  """Retrieves a Keras loss as a `function`/`Loss` class instance.
+
+  You can get loss as a `function` using parameter `identifier` as a string
+  of the loss function as shown in below example.
+  >>> loss = tf.keras.losses.get("categorical_crossentropy")
```

Comment:
Please add an empty line before every doctest to not break formatting on the website